### PR TITLE
Features/error msg confirm prefs #159012870

### PR DESCRIPTION
**Pull request by fddayan**
https://www.pivotaltracker.com/story/show/159012870

In order to test this preview app, there is a failure hardcoded. before merging it has to be disabled
https://github.com/Exygy/sf-dahlia-lap/pull/81/files#diff-302abce137cdaf8d1b599d1fa4668ca4R16

### CODE REVIEW

**Comment by chrisdolendo** [comment]
@fddayan Please update this with the latest master which allows me to see the preference panel fields :)

**Comment by fddayan** [comment]
@chrisdolendo merged with master

File: app/controllers/api/v1/short_form_controller.rb [hunk 1]
```diff
@@ -3,14 +3,20 @@ class Api::V1::ShortFormController < ApiController
   before_action :authenticate_user!
 
   def submit
-    logger.debug "application_api_params: #{application_api_params}"
-    short_form_validator = ShortFormValidator.new(application_api_params)
-    if short_form_validator.valid?
-      application = application_service.submit(application_api_params)
-      logger.debug "application submit response: #{application}"
-      render json: { application: application }
+    # logger.debug "application_api_params: #{application_api_params}"
```
Comment (callaghanc) [inline app/controllers/api/v1/short_form_controller.rb]: @fddayan Our practice in the web app, which you haven't seen so I don't expect you to know it 🙂 has been to add a TODO comment whenever we add an error trigger for testing purposes. The format would be like: "TODO: Remove this testing error trigger after story acceptance. This is a trigger to [insert description of trigger here]." This helps us be totally clear and aware of what parts of a PR's code are for testing error triggers, and is an extra reminder to be sure to remove the error trigger before merge. Could you add a comment in that format here?

File: app/javascript/components/supplemental_application/sections/PreferencesTable.js [hunk 2]
```diff
@@ -70,19 +83,21 @@ const expanderAction = (row, expanded, expandedRowToggler) => {
...
-    <ExpandableTable
-      columns={columns}
-      rows={rows}
-      expanderRenderer={expanderAction}
-      expandedRowRenderer={expandedRowRenderer(application, applicationMembers, onSave)}
-    />
-  </TableWrapper>)
+  const rows = buildRows(application, fileBaseUrl)
+  return (
+    <React.Fragment>
```
Comment (callaghanc) [inline app/javascript/components/supplemental_application/sections/PreferencesTable.js]: @fddayan Do we need a Fragment here? Looks like it's wrapping just a single item.

Comment (fddayan) [reply app/javascript/components/supplemental_application/sections/PreferencesTable.js]: we do not

**Comment by fddayan** [comment]
@callaghanc ready

**Comment by fddayan** [comment]
Closing so that we can re-create after cleaning the repo of VCR cassette data.

**Pull request closed without merging**